index.bs

```diff
@@ -1683,10 +1683,12 @@ following additional fields:
     display could return this value as 1.6 to indicate its preferred content
     scaling. Default is none.
 
-: color-profiles (optional)
-:: An optional field indicating what color profiles are understood.
-    The listener may use these values to determine how to encode
-    video. Some examples include: sRGBv4, Rec709, DciP3. The default value is sRGBv4.
+: color-gamut (optional)
+:: An optional field indicating what color spaces can be understood by the media
+    receiver. The media sender may use these values to determine how to encode
+    video. Valid values correspond to
+    [[MEDIA-CAPABILITIES#colorgamut|ColorGamut]] in the Media Capabilities API.
+    The default value is "srgb".
```

Comment:
```suggestion
    The default value is "srgb". [[!MEDIA-CAPABILITIES]]
```

Use of Bikeshed's cross-spec section link is not enough to add Media Capabilities to the list of normative references. There needs to be an explicit bibliographical reference.